src/Connectors/VidaDocumentAiVerificationConnector.php

```diff
@@ -4,14 +4,19 @@
 
 namespace Ziming\LaravelVidaId\Connectors;
 
-use Saloon\Helpers\OAuth2\OAuthConfig;
 use Saloon\Http\Connector;
+use Illuminate\Support\Facades\Cache;
+use Saloon\Helpers\OAuth2\OAuthConfig;
+use Saloon\CachePlugin\Traits\HasCaching;
+use Saloon\CachePlugin\Contracts\Cacheable;
 use Saloon\Traits\OAuth2\ClientCredentialsGrant;
+use Saloon\CachePlugin\Drivers\LaravelCacheDriver;
 use Ziming\LaravelVidaId\Resources\DocumentAiResource;
 
-class VidaDocumentAiVerificationConnector extends Connector
+class VidaDocumentAiVerificationConnector extends Connector implements Cacheable
 {
     use ClientCredentialsGrant;
+    use HasCaching;
```

Comment:
hmm cache at the connector may be too risky for now cuz it will apply for all the requests that use this connector.

Can you apply the caching to the specific request classes that you feel could benefit instead?

Thanks!